packages/lablqml/lablqml.0.6.2/opam

```diff
@@ -1,11 +1,15 @@
 opam-version: "2.0"
 maintainer:   "kakadu.hafanana@gmail.com"
 authors:      "kakadu.hafanana@gmail.com"
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
 homepage:     "http://kakadu.github.io/lablqt/"
 bug-reports:  "https://github.com/kakadu/lablqml/issues"
 dev-repo: "git+https://github.com/Kakadu/lablqml.git"
 tags: [ "gui" "ui" "qt" ]
 
+# Marking as unavailable because on CI file system is readonly and configure fails
```

Comment:
Note that this is not the CI system but the sandbox in opam 2.1 that's not making it fail. The only reason why this worked in CI before is that the sandbox was not enabled and that we were testing only opam 2.0, but for users using 2.1 it was failing too.

Reply:
It looks like you don't insist on fixing this, but I rephrased that anyway.